### PR TITLE
treelist: add ctrl + l/r arrow, edit ctrl + u/d arrow; datagrid: edit ctrl + u/d arrow

### PR DESCRIPTION
**Pull request by pavel-2020**
[Trello card](https://trello.com/c/Q7w6dWMq/1487-add-description-for-keys-to-treelist-keyboard-navigation)

### CODE REVIEW

File: JSDemos/Demos/DataGrid/KeyboardNavigation/description.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 
-- **Tab**       
-Navigate between grid elements.
+- **Enter**
+Execute an action on a focused element.
 
-- **Ctrl** + **&uarr;** or **Ctrl** + **&darr;**        
-Navigate between header, filter panel, filter row, data area, and pager.
+- **Tab**
+Navigate between TreeList elements.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/DataGrid/KeyboardNavigation/description.md]:
```suggestion
Navigate within DataGrid elements.
```

File: JSDemos/Demos/TreeList/KeyboardNavigation/description.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
 In this demo, you can use the following keys and key combinations to interact with the UI component:
 
+- **Enter**
+Execute an action on a focused element.
+
 - **Tab**
 Navigate between TreeList elements.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/TreeList/KeyboardNavigation/description.md]:
```suggestion
Navigate within TreeList elements.
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]
In both lists, item headers and item descriptions are displayed in one line. Don't forget about 4 or more space characters after the item header.

**Review by RomanTsukanov — DISMISSED** [review]

File: JSDemos/Demos/DataGrid/KeyboardNavigation/description.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
-- **Ctrl** + **&uarr;** or **Ctrl** + **&darr;**        
-Navigate between header, filter panel, filter row, data area, and pager.
+- **Tab**                
+Navigate within DataGrid elements.
 
-- **Enter** or **Space**        
-Execute an action on a focused element.
\ No newline at end of file
+- **Ctrl** + **&uarr;** or **Ctrl** + **&darr;**            
+Navigate between column header, filter row, data area, filter panel, and pager.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/DataGrid/KeyboardNavigation/description.md]:
```suggestion
Navigate between a column header, filter row, data area, filter panel, and pager.
```

File: JSDemos/Demos/TreeList/KeyboardNavigation/description.md [hunk 4]
```diff
@@ -1,10 +1,13 @@
...
 
-- **Ctrl** + **&uarr;** or **Ctrl** + **&darr;**
-Navigate between header, filter panel, filter row, data area, and pager.
+- **Tab**             
+Navigate within TreeList elements.
 
-- **Enter** or **Space**
-Execute an action on a focused element.
+- **Ctrl** + **&uarr;** or **Ctrl** + **&darr;**             
+Navigate between column header, filter row, data area, filter panel, and pager.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/TreeList/KeyboardNavigation/description.md]:
```suggestion
Navigate between a column header, filter row, data area, filter panel, and pager.
```

**Review by dirk-pieterse — DISMISSED** [review]

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**